src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Instead of using 'else if', just use another 'if' statement with a 'return' since the previous 'if' condition also contains a 'return'. This simplifies the logic and makes it clearer that the conditions are mutually exclusive.